### DISCUSSION TITLE
toolbar color change

### DISCUSSION THREAD

**Discussion by sohailamjad256** [excalidraw/excalidraw#4615] [Q&A]
hi..can we change excalidraw  toolbar color? inplace of purple can we give different color

**Comment by dwelle** [comment]
Yep! Please refer to [this](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README_NEXT.md#customizing-styles) (unreleased readme).

**Reply by sohailamjad256** [reply]
![toolbar bg color](https://user-images.githubusercontent.com/53993286/149831779-dbd9d960-3138-4c76-90f5-017c57cb4fb0.png)
Hi i have attached one toolbar image ...now its coming in slateblue color..so, i just want change that bg color..if you send me the exact code where i do need to change ..it will be very helpfull..thanks

**Reply by ad1992** [reply]
You mean the purple color right ? could you share a code sandbox example how you are doing it ?

**Reply by sohailamjad256** [reply]
yes..purple color i just want to change in different color

**Reply by sohailamjad256** [reply]
where ever purple color is showing i want to replace with my color

**Reply by sohailamjad256** [reply]
actually its urgent can you give solution for that please

**Reply by ad1992** [reply]
I am not sure what you are doing hence I asked for the code, you just need to update `--primaryColor` as mentioned in the link,  you can refer to the [code sandbox example](https://1m7pt.csb.app/)

**Reply by sohailamjad256** [reply]
got it @ad1992 ..Thankyou
